### ISSUE TITLE
Creating new window from nvim-tree breaks default window settings

### ISSUE THREAD

**Issue by vhyrro** [kyazdani42/nvim-tree.lua#290]
I have these settings set for tabs:
```lua
vim.bo.expandtab = true
vim.bo.copyindent = true
vim.bo.shiftwidth = 4
vim.bo.tabstop = 4
vim.bo.softtabstop = 4
vim.bo.autoindent = true
vim.bo.smartindent = true
```
Whenever I open a file using `:e` or just `nvim file` my tab settings are respected, yet whenever I open a file through nvim-tree (the `o` keybinding), my tab settings seem to be erased or overwritten only for that file. Having to work around this using autocommands is rather tedious :)

**Comment by Vhyrro** [comment]
After investigating a bit more, it seems to be a neovim internal problem, as this plugin's code does not seem to do any modification in that regard

The only way to fix it seems to be to set up an autocommand

```lua
vim.cmd [[ autocmd FileType * :lua setup_tabs() ]]
```

Closing, but maybe double checking wouldn't hurt just in case :)

**Comment by kyazdani42** [comment]
i will investigate soon anyway, maybe this is related to the way the plugin is creating it's windows/buffer

**Comment by kyazdani42** [comment]
humm i've seen that issue too, i might need help because it's an area that i cannot find documentation for. 
But for what i remember, vim/nvim will use current window settings when opening another window. this is quite confusing in this situation.

**Comment by kyazdani42** [comment]
should be fixed in 4388338a6a1
